tf-asm/src/main/java/twilightforest/asm/transformers/player/GetFieldOfViewModifierTransformer.java

```diff
@@ -0,0 +1,66 @@
+package twilightforest.asm.transformers.player;
+
+import cpw.mods.modlauncher.api.ITransformer;
+import cpw.mods.modlauncher.api.ITransformerVotingContext;
+import cpw.mods.modlauncher.api.TargetType;
+import cpw.mods.modlauncher.api.TransformerVoteResult;
+import net.neoforged.coremod.api.ASMAPI;
+import org.jetbrains.annotations.NotNull;
+import org.objectweb.asm.Opcodes;
+import org.objectweb.asm.tree.*;
+import twilightforest.asm.ASMUtil;
+
+import java.util.Set;
+
+/**
+ * {@link twilightforest.asmhooks.PlayerHooks#forwardBoostNullify}
+ * {@link twilightforest.asmhooks.PlayerHooks#forwardBoostRestore}
+ */
+public class GetFieldOfViewModifierTransformer implements ITransformer<MethodNode> {
+
+	@Override
+	public @NotNull TargetType<MethodNode> getTargetType() {
+		return TargetType.METHOD;
+	}
+
+	@Override
+	public @NotNull MethodNode transform(MethodNode node, ITransformerVotingContext context) {
+		node.instructions.insert(ASMAPI.listOf(
```

Comment:
Why is this inserting at HEAD? That's above even the first label node. Should be a bit more specific with the injection point, when does Nullify actually need to be invoked?

Reply:
we should not be injecting above even label nodes